CONTRIBUTING.md

```diff
@@ -20,11 +20,12 @@ The animations in Animate.css should follow a few key principles:
 1. Match the naming convention (camelCase, categorized [fades, bounces, etc])
 2. Indent with two spaces
 3. End each file with a blank line
+4. Please use the configurations in .editorconfig and .prettierrc to help keeps styling consistant.
```

Comment:
I think something like this would work better:
> Make sure you have an editorconfig plugin/extension enabled in your editor and all the dependencies installed so prettier can automatically format your code when committing.

Reply:
I implemented this change.